lib/ui/widgets/password_text_field.dart

```diff
@@ -55,4 +57,14 @@ class _PasswordFormFieldState extends State<PasswordFormField> {
       _obscureText = !_obscureText;
     });
   }
+
+  Color getErrorTextColor() {
+    return widget.appTheme == ThemeMode.light.toString()
+        ? Colors.amberAccent
+        : widget.appTheme == ThemeMode.dark.toString()
+            ? Colors.redAccent
+            : Theme.of(context).brightness == Brightness.light
+                ? Colors.amberAccent
+                : Colors.redAccent;
+  }
```

Comment:
```suggestion

  Color get errorTextColor => Theme.of(context).brightness == Brightness.light
                ? Colors.amberAccent
                : Colors.redAccent;
```

Same as login_view.dart